test/run-php-linter.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```

Comment:
Sure I'll do that.
If we want to take the plunge we could use the Docker based testing in Travis too. I thought that might be too much change for one PR though.